R/compute_clusters.R

```diff
@@ -25,7 +25,7 @@ compute_clusters <- function(feature_tables,
                              mz_tol_absolute,
                              mz_max_diff,
                              rt_tol_relative,
-                             do.plot = FALSE,
+                             do.plot,
```

Comment:
Maybe this could at some point be outsourced to be an option of the package, something like a global variable.

https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/options